assets/src/edit-story/masks/frame.js

```diff
@@ -63,6 +63,26 @@ const DropTargetPath = styled.path`
   transition: opacity 0.5s;
   pointer-events: visibleStroke;
   opacity: ${({ active }) => (active ? 0.3 : 0)};
+  stroke: ${({ theme }) => theme.colors.border.selection};
+`;
+
+const Wrapper = styled.div`
+  width: 100%;
+  height: 100%;
+
+  ::before {
+    transition: opacity 0.5s;
+    opacity: ${({ hasBackgroundOutline }) => (hasBackgroundOutline ? 1 : 0)};
+    display: block;
+    content: '';
+    position: absolute;
+    width: calc(100% + 8px);
+    height: calc(100% + 8px);
+    border-radius: 8px;
+    left: -4px;
+    top: -4px;
```

Comment:
These would be nice to have as a constant, too -- 4 being the gap and 8 being 2 times the gap.